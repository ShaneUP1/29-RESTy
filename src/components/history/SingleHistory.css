.SingleHistory {
  p {
    line-height: 2px;
  }
}
