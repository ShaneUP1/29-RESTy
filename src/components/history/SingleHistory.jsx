import React from 'react';
import PropTypes from 'prop-types';
import styles from './SingleHistory.css';

const SingleHistory = ({ url, method }) => (
  <div className={styles.SingleHistory} >
    <p>{url}</p>
    <p>{method}</p>
  </div>
);

SingleHistory.propTypes = {
  url: PropTypes.string.isRequired,
  method: PropTypes.string.isRequired
};

export default SingleHistory;
