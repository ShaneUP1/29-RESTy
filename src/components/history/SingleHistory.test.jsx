import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SingleHistory from './SingleHistory';


describe('SinlgeHistory component', () => {
  afterEach(() => cleanup());
  it('renders a list of histories', () => {
    const { asFragment } = render(<SingleHistory
      url={"www.url.com"} method={"GET"}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
