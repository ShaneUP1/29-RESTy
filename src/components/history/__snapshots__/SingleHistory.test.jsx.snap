// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SinlgeHistory component renders a list of histories 1`] = `
<DocumentFragment>
  <div
    class="SingleHistory"
  >
    <p>
      www.url.com
    </p>
    <p>
      GET
    </p>
  </div>
</DocumentFragment>
`;
